_corporate-pages/information-security.markdown

```diff
@@ -0,0 +1,92 @@
+footer-order: "3"
+---
+
+Keeping your data and the data of your members secure is at the very core of the Join Together platform. From our infrastructure to our software design, through to our working practices, security and data safety is central to every thing we do.
+
+## How we take care of your data
+
+### ISO 27001 data centres
+
+Join Together’s infrastructure runs on EU-based servers provided by Heroku, which is built on Amazon Web Services (AWS). Heroku provides a Platform as a Service (PaaS) that comes with exceptional security out of the box.
+
+Heroku’s physical infrastructure is hosted and managed within Amazon’s secure data centres. Amazon continually manages risk and undergoes regular assessments to ensure compliance with industry standards.
+
+Amazon’s data centre operations have been accredited under ISO 27001, SOC 1 and SOC 2/SSAE 16/ISAE 3402 (Previously SAS 70 Type II), PCI Level 1, FISMA Moderate and Sarbanes-Oxley (SOX).
+
+Learn more about Heroku and AWS security:
+- [Heroku Security](https://www.heroku.com/policy/security)
+- [AWS Security](https://aws.amazon.com/security/)
+- [AWS Compliance](https://aws.amazon.com/compliance/)
+
+### Physical security
+
+We do not have dedicated data centres as our infrastructure is cloud-based. Physical security to the servers our platform runs on and to your data is managed by [AWS security certifications](https://aws.amazon.com/compliance/iso-27001-faqs/).
+
+### Data storage and encryption at rest
+
+Your data is [encrypted at rest](https://devcenter.heroku.com/articles/heroku-postgres-production-tier-technical-characterization#data-encryption) in our databases using AES-256 block-level storage encryption.
+
+### Passwords
+
+All user passwords are stored in our database using Bcrypt, a complex one-way password hashing algorithm that provides protection over common attacks such as rainbow tables and brute-force search attacks.
+
+### Encryption in transit
+
+All communication between you, users using your join form, and any integrations with your internal CRM systems traverse the Internet via encrypted HTTPS traffic using TLS v1.3 were supported. We support older clients with TLS v1.2.
+
+This encryption during communication ensures information cannot be read or manipulated by unauthorised third parties.
+
+### Backups and data retention
+
+We ensure that all data is regularly backed up. A daily backup is taken each day and retained for 7 days. Weekly backups are retained for 4 weeks.
+
+By default we redact all user-submitted personal data in incomplete or unsubmitted applications after 14 days. All user-submitted personal data in submitted applications is redacted 60 days after you have retrieved them for processing. If you feel that these retention periods do not suit you, you can choose to change them at any time.
```

Comment:
```suggestion
By default we redact all user-submitted personal data in incomplete or unsubmitted applications after 14 days. All user-submitted personal data in submitted applications is redacted 60 days after you have retrieved them for processing.
```

Removed "If you feel that these retention periods do not suit you, you can choose to change them at any time." – these are constants at present; do we have plans to make them configurable?

Reply:
We say this in our Data Protection Fact Sheet. I think the idea is that we could very easily make it configurable if these periods were suitable for a specific union.